app/config.py

```diff
@@ -1 +1,12 @@
+import os
+
 DEBUG = True
+
+DEFAULT_RENDERERS = ['app.helper.custom_api_render.JSONRenderer',
```

Comment:
This looks ugly as hell. Like java xml hibernate configuration level of ugly ;-) What does it do?